include/pfs/types.hpp

```diff
@@ -343,6 +343,12 @@ struct module
     bool operator<(const module& rhs) const { return name < rhs.name; }
 };
 
+struct uptime
+{
+    double running_time;
```

Comment:
According to the man page, the first value is: "the uptime of the system (including time spent in suspend)"
I think we should use the value "system_time" here.

Reply:
Done.